functional-tests.sh

```diff
@@ -77,6 +77,8 @@ if [ "$ENABLE_HTTPS" != "1" ]; then
 fi
 
 SERVER_ALIAS="myminio"
+SERVER_ALIAS_1="myminio-ssl"
```

Comment:
Name this as `SERVER_ALIAS_TLS` instead which can provide more meaning.

Reply:
done